src/localization/es.js

```diff
@@ -16,6 +17,8 @@ export default {
   gatherAction: 'Recolectar ',
   weighAction: 'Pesar ',
   baleAction: 'Enfardar ',
+  baleEditError: 'No se pudo guardar el fardo, pruebe nuevamente ',
```

Comment:
We have two strings for the same. Maybe change the text to be more descriptive or make the same string for both cases.